src/Content/ProjectsInfo.js

```diff
@@ -11,10 +11,10 @@ const ProjectsInfo = () => {
 
   const { data, isSuccess } = useGetProjectsQuery();
 
-  useEffect(async () => {
+  useEffect(() => {
     try {
       if (isSuccess) {
-        const projects = await JSON.parse(JSON.stringify(data));
```

Comment:
That was not the only place with those shenanigans, remove all other spots like that.